README.md

```diff
@@ -183,6 +183,28 @@ cmake --build . -j
 sudo make install
 ```
 
+> [!Note] 
+>
+> For systems like **KDE Linux** (https://community.kde.org/KDE_Linux) or other immutable distributions that lack tools such as `rpm-ostree`, it is necessary to manually create a user overlay to supplement missing or non-modifiable system paths.
+>
+> First, clone the source code and compile it into a user-owned directory:
+>
+> ```bash
+> git clone https://github.com/matinlotfali/KDE-Rounded-Corners
```

Comment:
Because this piece is repeated above, I think it is better not to rewrite it. I also have skipped it in the X11 build and only mentioned the difference.